### ISSUE TITLE
Rule Proposal: prefer-numeric-literals

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#6068]
From [requireNumericLiterals].

This rule will disallow `parseInt` with a constant value as the favor of numeric literals.

```js
{
    "prefer-numeric-literals": "error"
}
```

[requireNumericLiterals]: http://jscs.info/rule/requireNumericLiterals

**Comment by ilyavolodin** [comment]
Should this be an option for `radix` rule? I don't know if it's the right fit, but `radix` already deals with `parseInt`

**Comment by mysticatea** [comment]
`radix` rule focuses on the 2nd argument of `parseInt` to avoid a use of unintentional radix.
This proposal's rule focuses on whole `parseInt` to prefer hex/octal/binary numeric literals.
It's good that separating those 2 rules, IMO.

**Comment by nzakas** [comment]
Per TSC meeting (7/21/16), marking as accepted.

**Comment by hzoo** [comment]
I can probably take this one (made the JSCS rule originally)

**Comment by nzakas** [comment]
Sweet, thanks @hzoo!

**Comment by hzoo** [comment]
@azhang496 said she would take this (I'll guide/help)
